server/ctrlsubsonic/handlers_raw.go

```diff
@@ -118,14 +118,16 @@ func (c *Controller) ServeStream(w http.ResponseWriter, r *http.Request) *spec.R
 		play.Count++           // for getAlbumList?type=frequent
 		c.DB.Save(&play)
 	}()
-	client := params.GetOr("c", "*")
+	client := params.Get("c")
```

Comment:
i remembered that the `c` param is mandatory anyway, so this will never be `""`

Reply:
:+1: